releasenotes/notes/0.45/removed_deprecated_0.21-6c93f7bbc50ae40e.yaml

```diff
@@ -1,44 +1,48 @@
 ---
-upgrade:
+upgrade_misc:
   - |
     The function :func:`~qiskit.execute_function.execute`
-    does not accept the arguments `qobj_id` and  `qobj_header` any more.
-    Their use was deprecated in Qiskit 0.37 (with Terra 0.21), released on June 2022.    
+    does not accept the arguments ``qobj_id`` and  ``qobj_header`` anymore.
+    Their use was deprecated in Qiskit 0.37 (with Terra 0.21), released on June 2022.
+upgrade_transpiler:
   - |
-    The transpilation pass ``qiskit.transpiler.passes.ALAPSchedule`` is removed. It use was deprecated 
-    in Qiskit 0.37 (with Terra 0.21), released on June 2022 and replaced by 
+    The transpilation pass ``qiskit.transpiler.passes.ALAPSchedule`` is removed. Its use was deprecated
```

Comment:
Did we actually ever deprecate these old scheduling passes? I thought they were still pending deprecation warning. If so we should not be removing them and we'll need to restore them before release.

Reply:
Yeah we never deprecated it, so we need to restore it before we can release: https://qiskit.org/documentation/stubs/qiskit.transpiler.passes.ALAPSchedule.html#alapschedule

Reply:
I fixed this in: https://github.com/Qiskit/qiskit/pull/11184